webservices/common/views.py

```diff
@@ -132,3 +132,35 @@ def validate_kwargs(self, kwargs):
                 ),
                 status_code=422,
             )
+
+
+# return all results, per_page without cap (cap=0). It is used for:
+# ScheduleABySizeCandidateView
+# ScheduleAByStateCandidateView
+# ScheduleAByStateCandidateTotalsView
+class NoCapResource(utils.Resource):
+
+    args = {}
+    model = None
+    schema = None
+    page_schema = None
+    index_column = None
+    unique_column = None
+    filter_match_fields = []
+    filter_multi_fields = []
+    filter_range_fields = []
+    filter_fulltext_fields = []
+    filter_overlap_fields = []
+    query_options = []
+    join_columns = {}
+    aliases = {}
+    cap = ''
+
+    @use_kwargs(Ref('args'))
+    @marshal_with(Ref('page_schema'))
+    def get(self, *args, **kwargs):
+        query = self.build_query(*args, **kwargs)
+        multi = False
+        if isinstance(kwargs['sort'], (list, tuple)):
+            multi = True
+        return utils.fetch_page(query, kwargs, multi=multi, cap=0)
```

Comment:
This is really great work Jun! You could pass in `is_count_exact=True` to `utils.fetch_page` to indicate that all the counts are exact for this resource so that it doesn't say null.

Reply:
![image](https://github.com/fecgov/openFEC/assets/121631201/3f678af5-4e86-4860-b9b7-6fae911d0a1f)